server/services/datacommons.py

```diff
@@ -172,24 +172,21 @@ def obs_series_within(parent_entity, child_type, variables):
       })
 
 
-def v2observation(select, entity, variable):
```

Comment:
There are use cases to pass in the expression. All the functions in datacommons.py should be in the exact form of V2 API arguments. Wrapping can be done in fetch.py.

Reply:
I see, reverted.